modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/hostobject/HostObjectTestCase.java

```diff
@@ -189,8 +189,8 @@ public void testHostObjectTestCase() throws Exception {
         SubscriptionRequest subscriptionRequest = new SubscriptionRequest(APIName, providerName);
         subscriptionRequest.setApplicationName("HostObjectTestAPI-Application");
         apiStore.subscribe(subscriptionRequest);
-        apiPublisher.addDocument(APIName, APIVersion, providerName, "Doc-Name", "How To", "In-line",
-                                 "url-no-need", "summary", "");
+        apiPublisher.addDocument(APIName, APIVersion, providerName, "Doc-Name", "How To", "Inline", "url-no-need",
```

Comment:
fixed in commit: https://github.com/wso2/product-apim/commit/d4adc2898db0ba29784d1a4a223711ce0a51d3a2